.gitignore

```diff
@@ -3,7 +3,5 @@ node_modules
 ngrok
 package.json
```

Comment:
`package.son` definitely should not be in the `.gitignore` file. You will want to track any changes to this file. Not sure how this line got there. If our project generator put that there, that's a problem with that package.